extensions/quartz/runtime/src/main/java/io/quarkus/quartz/runtime/QuartzExtensionPointConfig.java

```diff
@@ -18,5 +18,6 @@
      * The properties passed to the class.
      */
     @ConfigDocMapKey("property-name")
+    @ConfigItem(name = ConfigItem.PARENT)
```

Comment:
Hmmm, no, this should stay the way it is. That's what is documented.

Reply:
Ah right, good spot. I do not remember why we went this way and did not double check (thanks for doing it), so seeing the warming I thought something was off. Do you remember the reasoning?

Reply:
I'll revert this change and only fix the warnings. The configuration knobs that users are seeing at the moment seems off - I think this is a worth improvement, let's reconsider it separately? WDYT @gsmet ?

Reply:
What is off exactly? Not sure I understand?

Reply:
I do not recall why we need `properties` in the keys:  `quarkus.quartz.[plugins/listeners]."name".properties."property"` e.g `quarkus.quartz.plugins.jobHistory.properties.jobSuccessMessage` instead of `quarkus.quartz.plugins.jobHistory.jobSuccessMessage`.

Reply:
The main reason I did it that way is that I didn't want a bag of random properties at the root.

Reply:
> The main reason I did it that way is that I didn't want a bag of random properties at the root.

Okay, thanks for refilling my memory. 

But since the root i.e `quarkus.quartz.plugins.jobHistory` (for example), can only be followed by properties, I think it would have been okay to keep the configuration know without the `.proprties.` segment. But the ship has sailed now.